test/index.spec.js

```diff
@@ -2,16 +2,99 @@ const chai = require('chai');
 const nock = require('nock');
 const { getTestServer } = require('./test-server');
 
+const userPayload = require('./payload/user');
+
 chai.should();
 
+// nock.disableNetConnect();
```

Comment:
once you have tests stable-ish, re-enable this and adjust as needed to make it work